src/content/guides/production.mdx

```diff
@@ -165,7 +165,7 @@ Many libraries will key off the `process.env.NODE_ENV` variable to determine wha
   });
 ```
 
-T> Technically, `NODE_ENV` is a system environment variable that Node.js exposes into running scripts. It is used by convention to determine dev-vs-prod behavior by server tools, build scripts, and client-side libraries. Contrary to expectations, `process.env.NODE_ENV` is not set to `'production'` **within** the build script `webpack.config.js`, see [#2537](https://github.com/webpack/webpack/issues/2537). Thus, conditionals like `process.env.NODE_ENV === 'production' ? '[name].[contenthash].bundle.js' : '[name].bundle.js'` within webpack configurations do not work as expected.
+T> Technically, `NODE_ENV` is a system environment variable that Node.js exposes into running scripts. It is used by convention to determine dev-vs-prod behavior by server tools, build scripts, and client-side libraries.
```

Comment:
This guide doesn't mention the usage of `--node-env`.

Reply:
Indeed, seems I was wrong to assume it's related to `--node-env`, description updated. But still it's stale content as there's no `-p` anymore [from what I can tell](https://github.com/webpack/webpack-cli/pull/1693)? Hence it doesn't make sense to link to it.

Reply:
Thanks for heads-up, absolutely my oversight. I've went over that thread, and think we can still make some improvements here. Let me know what you think.